src/components/Uploader/style.scss

```diff
@@ -22,19 +22,18 @@
 }
 
 .instructionsCopy {
-  margin: 14*@unit-small 24*@unit-small;
-  font-size: @font-size-small;
+  margin: 14*$unit-small 24*$unit-small;
+  font-size: $font-size-small;
   line-height: 1.43;
 
   @media (--small-viewport) {
-    font-size: @font-size-base;
-    margin: 14*@unit 24*@unit;
+    font-size: $font-size-base;
+    margin: 14*$unit 24*$unit;
   }
 }
 
 .error {
-  &:extend(.text);
```

Comment:
You no longer need this?

Reply:
I checked against the existing one & how the UI looks in Surge Latest, that class doesn't exist in the stylesheet & the screen itself doesn't have anything with that class generated for the errors. Might have been leftover code.